adele-booking/src/main/java/com/microservicesteam/adele/model/User.java

```diff
@@ -7,4 +7,12 @@
 
     Long id();
```

Comment:
Yes, and actually it won't have any in the future either, just a token which identifies it in FB, Google, Github or whatever authentication service we gonna use